package.json

```diff
@@ -91,6 +91,9 @@
     "@humanwhocodes/module-importer": "^1.0.1",
     "@humanwhocodes/retry": "^0.3.0",
     "@nodelib/fs.walk": "^1.2.8",
+    "@types/estree": "^1.0.6",
+    "@eslint/core": "^0.6.0",
+    "@types/json-schema": "^7.0.15",
```

Comment:
Shouldn't they be sorted alphabetically? So `@eslint/core` should be after `@eslint/config-array`?